examples/README.md

```diff
@@ -8,7 +8,11 @@ Examples on how to deploy HiveMQ Platform clusters with the HiveMQ Platform Oper
 
 ### [hivemq-operator](./hivemq-operator)
 
-Examples for the legacy HiveMQ Operator
+Examples for the legacy HiveMQ Operator chart
+
+### [hivemq-swarm](./hivemq-swarm)
+
+Includes an example about how to use and install a Prometheus ServiceMonitor along with a custom Grafana dashboard through a ConfigMap for the HiveMQ Swarm chart
 
 ## Documentation
 See the [HiveMQ Kubernetes documentation](https://docs.hivemq.com/hivemq-platform-operator/introduction.html) for more detailed information.
```

Comment:
Lets remove this last documentation section and instead add a documentation link to each section above.